### DISCUSSION TITLE
Rotating file handle for pytest logging

### DISCUSSION THREAD

**Discussion by jay746** [pytest-dev/pytest#8580] [Ideas]
Is this possible to give option in pytest logging for creating a new file once the given size of file created?

**Comment by RonnyPfannschmidt** [comment]
Pytest itself doesn't deal with log files, so it's all up to you

**Reply by jay746** [reply]
It does isn't .. i see doc @https://docs.pytest.org/en/stable/logging.html

Yeah logging is done by logger module but capturing and keeping in file taken care by pytest command
So shouldn't be rotating file be taken care by pytest.

If not can you suggest how to take care with pytest

**Reply by RonnyPfannschmidt** [reply]
pytest does not create a logfile, its in memory
